src/diagnostics.rs

```diff
@@ -32,6 +39,8 @@ impl fmt::Display for TerminationInfo {
                 write!(f, "{}", msg),
             Deadlock =>
                 write!(f, "the evaluated program deadlocked"),
+            MultipleSymbolDefinitions { link_name, .. } =>
+                write!(f, "{}", link_name)
```

Comment:
This should give a rough summary of what happened... I doubt `link_name` is that?
```rust
write!(f, "symbol `{}` defined twice", link_name)
```
could work